vello/src/render.rs

```diff
@@ -178,9 +178,9 @@ impl Render {
             // is zero.
             packed.resize(size_of::<u32>(), u8::MAX);
         }
-        let scene_buf = ResourceProxy::Buffer(recording.upload("scene", packed));
+        let scene_buf = ResourceProxy::Buffer(recording.upload("vello.scene", packed));
         let config_buf = ResourceProxy::Buffer(
-            recording.upload_uniform("config", bytemuck::bytes_of(&cpu_config.gpu)),
+            recording.upload_uniform("vello.config", bytemuck::bytes_of(&cpu_config.gpu)),
         );
         let info_bin_data_buf = ResourceProxy::new_buf(
             buffer_sizes.bin_data.size_in_bytes() as u64,
```

Comment:
There are lots of other calls to `new_buf`; why have you changed only the ones you have?

Reply:
because I wasn't looking at new_buf but `upload` / `upload_uniforms` ... I'll hit the others now, I guess. Thanks!

Reply:
Fixed and force pushed.